mk/artifact.mk

```diff
@@ -0,0 +1,57 @@
+USE_PREBUILT ?= 1
+
+CC ?= gcc
+CROSS_COMPILE ?= riscv-none-elf-
+
+BINDIR := $(abspath $(OUT))
+
+TEST_SUITES += \
+    ansibench \
+	rv8-bench
+
+# "ieee754" needs F extension
+# "smolnes", "ticks" have inline assembly and only work in riscv
+TESTBENCHES += \
+    captcha \
+    donut \
+    fcalc \
+    hamilton \
+    $(ieee754) \
```

Comment:
Use capitalized identifiers such as `PROG_ieee754` or `PROG_IEEE754`

Reply:
The testbenches quoted with `$(...)` are provided by `rv32emu` repository and are just a memo for the migration in the future. Should we simply remove them?